gradle/libs.versions.toml

```diff
@@ -17,28 +19,47 @@ detektFormatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", v
 kotlinStdlib = { module = "org.jetbrains.kotlin:kotlin-stdlib", version.ref = "kotlin" }
 espressoCore = { module = "androidx.test.espresso:espresso-core", version.ref = "espresso" }
 espressoWeb = { module = "androidx.test.espresso:espresso-web", version.ref = "espresso" }
-testCore = "androidx.test:core:1.3.0"
+testCore = "androidx.test:core:1.4.0"
 uiAutomator = "androidx.test.uiautomator:uiautomator:2.2.0"
-fragmentTesting = "androidx.fragment:fragment-testing:1.3.2"
+fragmentTesting = "androidx.fragment:fragment-testing:1.3.6"
+# upgrading to 4.6.1 is blocked by https://github.com/robolectric/robolectric/issues/6593
```

Comment:
It's not a bug, there is a solution in the comments

Reply:
I am not sure that you are right. Ok, can you try to fix it?